drivers/ieee802154/ieee802154_nrf5.c

```diff
@@ -441,7 +441,7 @@ NET_DEVICE_INIT(nrf5_154_radio, CONFIG_IEEE802154_NRF5_DRV_NAME,
 		nrf5_init, &nrf5_data, &nrf5_radio_cfg,
 		CONFIG_IEEE802154_NRF5_INIT_PRIO,
 		&nrf5_radio_api, OPENTHREAD_L2,
-		NET_L2_GET_CTX_TYPE(OPENTHREAD_L2), 125);
+		NET_L2_GET_CTX_TYPE(OPENTHREAD_L2), 1280);
```

Comment:
why is this change in the 'subsys: net: l2: Add OpenThread L2' commit?

Reply:
Not sure how it got here, I will clean commits after CMake porting is finished.